src/App.js

```diff
@@ -9,32 +10,39 @@ import { BrowserRouter as Router } from "react-router-dom";
  * Footer: Global Footer
  * Navbar: Global navigation bar
  */
-import {
-  Footer,
-  Navbar,
-} from "./components/molecules";
-// Routes
+import { Footer, Navbar } from "./components/molecules";
+// Starts the page on top when reloaded or redirected
+import { ScrollToTop } from "./components/atoms";
+
+//> Routes
 import Routes from "./Routes";
+//#endregion
 
+//#region > Components
 class App extends React.Component {
   render() {
     return (
       <Router>
```

Comment:
I would recommend to move the Router to index.js. If you disagree leave it as it is.
https://github.com/snek-at/front/issues/54